orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -323,37 +350,56 @@ def _anonymise_study_instances(
     """
     config = load_project_config(project_name)
     anonymised_instances_bytes = []
-    logger.debug("Zipped study infolist: {}", zipped_study.infolist())
+    logger.info("Processing project '{}', study: {}", project_name, study_uid)
+    skipped_instance_counts = defaultdict(int)
+    dicom_validation_errors = {}
+
     for file_info in zipped_study.infolist():
         with zipped_study.open(file_info) as file:
             logger.debug("Reading file {}", file)
             dataset = dcmread(file)
-
-            logger.info("Anonymising file: {} for study: {}", file, study_uid)
             try:
-                anonymised_instances_bytes.append(_anonymise_dicom_instance(dataset, config))
+                anonymised_instance, instance_validation_errors = _anonymise_dicom_instance(
+                    dataset, config
+                )
             except PixlSkipInstanceError as e:
-                logger.warning(
+                logger.debug(
                     "Skipping instance {} for study {}: {}",
                     dataset[0x0008, 0x0018].value,
                     study_uid,
                     e,
                 )
+                skipped_instance_counts[str(e)] += 1
             else:
+                anonymised_instances_bytes.append(anonymised_instance)
                 anonymised_study_uid = dataset[0x0020, 0x000D].value
+                dicom_validation_errors |= instance_validation_errors
+
+    if skipped_instance_counts:
+        logger.info(
+            "Project '{}' Study {}, skipped instance counts: {}",
+            project_name,
+            study_uid,
+            dict(skipped_instance_counts),
+        )
```

Comment:
Is this useful here, or should we just report it in the discard error below?

Reply:
yeah maybe in the discard error would be good, or make it debug rather than info?